lib/spoom/location.rb

```diff
@@ -73,7 +73,19 @@ def include?(other)
     def <=>(other)
       return unless Location === other
 
-      to_s <=> other.to_s
+      res = file <=> other.file
```

Comment:
```
[file, start_line, start_column, end_line, end_column] <=>
        [other.file, other.start_line, other.start_column, other.end_line, other.end_column]
```

Reply:
I like the suggestion. Easier to read and maintain this way.

> But on further thought, aren't end_line and end_column irrelevant here?

I am under the assumption we still want to compare if the the `end_line` or `end_column` are different. But I may be missing the big picture. Could you elaborate?

Reply:
I suspect that when we sort or compare locations, we're really interested in where it _begins_, i.e. the `file`, `start_line` and `start_colum`. May be worth looking into.